src/backends.jl

```diff
@@ -416,6 +442,10 @@ function _initialize_backend(pkg::PlotlyBackend)
         end
         _check_compat(PlotlyBase)
         _check_compat(PlotlyKaleido)
+        @eval begin
+            const PlotlyBase = Main.PlotlyBase
+            const PlotlyKaleido = Main.PlotlyKaleido
+        end
```

Comment:
what is that for?

Reply:
So currently, a package is imported in `Main` (hack for not adding it as a hard dependency in `Project.toml`). However, for some packages, you need to access the module itself (e.g. https://github.com/JuliaPlots/Plots.jl/blob/4fff117f5587abe30361ec367346752f10c5cbe6/src/backends/pyplot.jl#L12). We import `GR` directly in plots, but other backends need this. And for consistency, I've added this for all backends (here plotly).

Reply:
I wonder how well this works in Pluto (which does not evaluate in Main). But if I understood you right, if it worked before, it should still work, right?

Reply:
It should. Also, we have to distinguish the precompilation phase (which occurs before `__init__` where `@require` quicks in).